Cargo.toml

```diff
@@ -13,3 +13,11 @@ members = [
 
 [profile.dev]
 opt-level = 2
+
+[patch.crates-io.crypto-bigint]
+git = "https://github.com/risc0/RustCrypto-crypto-bigint"
+branch = "victor/use-risc0-accel" # DO NOT MERGE: Use a tag instead.
```

Comment:
Only change to take this out of draft status is to merge https://github.com/risc0/RustCrypto-crypto-bigint/pull/1 so that this can point to a tag instead. All the content is ready for review.

Reply:
Done https://github.com/risc0/RustCrypto-elliptic-curves/pull/1/commits/39f713e80d4c24bc6747319dbceab4a9d2083509